Randoop for Java version "4.3.0, local changes, branch master, commit 7af20ae, 2022-01-31".
Cannot instantiate non-accessible class com.google.javascript.rhino.head.Node$PropListItem specified via --testclass or --classlist; will use its static methods.

Will try to generate tests for 26 classes.
PUBLIC MEMBERS=596
Explorer = ForwardGenerator(steps: 0, null steps: 0, num_sequences_generated: 0;
    allSequences: 0, regresson seqs: 0, error seqs: 0=0=0, invalid seqs: 0, subsumed_sequences: 0, num_failed_output_test: 0;
    runtimePrimitivesSeen:38)

Progress update: steps=1, test inputs generated=0, failing inputs=0      (2022-02-23T04:32:08.970Z     355M used)
Progress update: steps=1000, test inputs generated=765, failing inputs=1      (2022-02-23T04:32:15.236Z     645M used)createAndClassifySequences threw an exception
randoop.main.RandoopBug: code=Call to public com.google.javascript.rhino.head.Scriptable com.google.javascript.rhino.head.Context.newArray(com.google.javascript.rhino.head.Scriptable,int) receiver: com.google.javascript.rhino.head.Context@b93aad args: [[object Object], -1] failed to run
	at randoop.util.ReflectionExecutor.executeReflectionCodeUnThreaded(ReflectionExecutor.java:169)
	at randoop.util.ReflectionExecutor.executeReflectionCode(ReflectionExecutor.java:94)
	at randoop.operation.MethodCall.execute(MethodCall.java:176)
	at randoop.operation.TypedOperation.execute(TypedOperation.java:310)
	at randoop.sequence.Statement.execute(Statement.java:163)
	at randoop.sequence.ExecutableSequence.executeStatement(ExecutableSequence.java:416)
	at randoop.sequence.ExecutableSequence.execute(ExecutableSequence.java:308)
	at randoop.sequence.ExecutableSequence.execute(ExecutableSequence.java:237)
	at randoop.generation.ForwardGenerator.step(ForwardGenerator.java:237)
	at randoop.generation.AbstractGenerator.createAndClassifySequences(AbstractGenerator.java:321)
	at randoop.main.GenTests.handle(GenTests.java:510)
	at randoop.main.Main.nonStaticMain(Main.java:71)
	at randoop.main.Main.main(Main.java:31)
Caused by: randoop.util.ReflectionCode$ReflectionCodeException: error invoking public com.google.javascript.rhino.head.Scriptable com.google.javascript.rhino.head.Context.newArray(com.google.javascript.rhino.head.Scriptable,int) on 3 arguments:
  com.google.javascript.rhino.head.Context@b93aad
  [object Object]
  -1
	at randoop.util.MethodReflectionCode.runReflectionCodeRaw(MethodReflectionCode.java:83)
	at randoop.util.ReflectionCode.runReflectionCode(ReflectionCode.java:59)
	at randoop.util.ReflectionExecutor.executeReflectionCodeUnThreaded(ReflectionExecutor.java:164)
	... 12 more
Caused by: java.lang.IndexOutOfBoundsException: Index: 0
	at com.google.javascript.rhino.head.NativeArray.listIterator(NativeArray.java:1806)
	at com.google.javascript.rhino.head.NativeArray.iterator(NativeArray.java:1791)
	at randoop.org.plumelib.util.StringsPlume.listToString(StringsPlume.java:1068)
	at randoop.org.plumelib.util.StringsPlume.listToStringAndClass(StringsPlume.java:1047)
	at randoop.org.plumelib.util.StringsPlume.toStringAndClass(StringsPlume.java:1022)
	at randoop.org.plumelib.util.StringsPlume.toStringAndClass(StringsPlume.java:999)
	at randoop.util.MethodReflectionCode.runReflectionCodeRaw(MethodReflectionCode.java:57)
	... 14 more


Randoop failed in an unexpected way.
Please report at https://github.com/randoop/randoop/issues ,
providing the information requested at https://randoop.github.io/randoop/manual/index.html#bug-reporting .
randoop.main.RandoopBug: code=Call to public com.google.javascript.rhino.head.Scriptable com.google.javascript.rhino.head.Context.newArray(com.google.javascript.rhino.head.Scriptable,int) receiver: com.google.javascript.rhino.head.Context@b93aad args: [[object Object], -1] failed to run
	at randoop.util.ReflectionExecutor.executeReflectionCodeUnThreaded(ReflectionExecutor.java:169)
	at randoop.util.ReflectionExecutor.executeReflectionCode(ReflectionExecutor.java:94)
	at randoop.operation.MethodCall.execute(MethodCall.java:176)
	at randoop.operation.TypedOperation.execute(TypedOperation.java:310)
	at randoop.sequence.Statement.execute(Statement.java:163)
	at randoop.sequence.ExecutableSequence.executeStatement(ExecutableSequence.java:416)
	at randoop.sequence.ExecutableSequence.execute(ExecutableSequence.java:308)
	at randoop.sequence.ExecutableSequence.execute(ExecutableSequence.java:237)
	at randoop.generation.ForwardGenerator.step(ForwardGenerator.java:237)
	at randoop.generation.AbstractGenerator.createAndClassifySequences(AbstractGenerator.java:321)
	at randoop.main.GenTests.handle(GenTests.java:510)
	at randoop.main.Main.nonStaticMain(Main.java:71)
	at randoop.main.Main.main(Main.java:31)
Caused by: randoop.util.ReflectionCode$ReflectionCodeException: error invoking public com.google.javascript.rhino.head.Scriptable com.google.javascript.rhino.head.Context.newArray(com.google.javascript.rhino.head.Scriptable,int) on 3 arguments:
  com.google.javascript.rhino.head.Context@b93aad
  [object Object]
  -1
	at randoop.util.MethodReflectionCode.runReflectionCodeRaw(MethodReflectionCode.java:83)
	at randoop.util.ReflectionCode.runReflectionCode(ReflectionCode.java:59)
	at randoop.util.ReflectionExecutor.executeReflectionCodeUnThreaded(ReflectionExecutor.java:164)
	... 12 more
Caused by: java.lang.IndexOutOfBoundsException: Index: 0
	at com.google.javascript.rhino.head.NativeArray.listIterator(NativeArray.java:1806)
	at com.google.javascript.rhino.head.NativeArray.iterator(NativeArray.java:1791)
	at randoop.org.plumelib.util.StringsPlume.listToString(StringsPlume.java:1068)
	at randoop.org.plumelib.util.StringsPlume.listToStringAndClass(StringsPlume.java:1047)
	at randoop.org.plumelib.util.StringsPlume.toStringAndClass(StringsPlume.java:1022)
	at randoop.org.plumelib.util.StringsPlume.toStringAndClass(StringsPlume.java:999)
	at randoop.util.MethodReflectionCode.runReflectionCodeRaw(MethodReflectionCode.java:57)
	... 14 more
